introduction.md

```diff
@@ -50,6 +50,26 @@ Below are the type of questions phinetics works on.
 
 * Are computers and robots morally responsible for their actions? 
 
+### Autonomous Machines
+
+* Can software possess autonomy?
+
+* Is autonomy a subjective or objective quality? 
+
+* Is there a need to change our definition of autonomy?
+
+### Memetics: replication and curation
+
+* How can the intellectual framework of memetics be applied to understand modern information technology?
```

Comment:
Memetics' fundamental premise is that: 
> Cultural evolution, including the evolution of knowledge, can be modelled through the same basic principles of variation and selection that underly biological evolution. 

`-` [Principia Cybernetica](http://pespmc1.vub.ac.be/memes.html)

Memetics is suspicious. There is an empirical basis for biological selection: genes can be singled out and observed, changes in a population's genotype have been observed. What is the empirical basis for memetics? A meme can be _any_ information pattern, so there is no way to reliably distinguish two similar memes. 

And even if we assume there is such a way, there is no way to observe the meme replicating itself _inside people's heads_. A computer can tell you that a YouTube video has been shared a million times, but cannot tell you whether or not, or how, the video is stored in people's minds.

The intellectual framework of memetics _is_ the analogy with biological evolution. How can biological evolution help us understand IT? Plainly, hardware and software are getting progressively more complex, efficient, and powerful. You could talk of certain technologies being _selected_ over competing technologies. But genes and technology are fundamentally different. Genes can be selected if they're the result of random mutations. What would be the corresponding observable phenomenon in technological evolution? Genes are inherited, and lineage/parenthood can be empirically traced. Again, what's the empirical equivalent in technology? 

So I don't think evolution's intellectual framework can shed much light on IT.